Cargo.toml

```diff
@@ -22,11 +22,13 @@ rustls-pemfile = "1.0.4"
 reqwest = { version = "0.12.4", features = ["json"] }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.116"
-chrono = { version = "0.4.38", features = ["serde"] }
+chrono = { version = "0.4.41", features = ["serde"] }
 dotenv = "0.15.0"
-arrow = "51.0.0"
-futures = "0.3.30"
+arrow = "55.1.0"
+futures = "0.3.31"
 base64 = "0.22.0"
+tracing = "0.1.41"
+spice-util = { git = "https://github.com/spiceai/spiceai", package = "util" }
```

Comment:
This is going to prevent us from publishing the SDK to crates.io - all dependencies must already be published on crates.